indra/llcommon/llapp.h

```diff
@@ -288,6 +290,8 @@ class LL_COMMON_API LLApp
     string_map mOptionMap;  // Contains all command-line options and arguments in a map
 
 protected:
+    virtual void forceQuit() {} // Puts the viewer into 'shutting down without error' mode.
+    virtual void requestQuit() {} // Request a quit. A kinder, gentler quit.
```

Comment:
I don't think the default behavior should be to return. I think if control actually reaches either of these base-class methods, it should call `std::exit()` or something comparable.

Reply:
> I don't think the default behavior should be to return

This behavior is not default, first I made these functions pure virtual,
but tests had failed due to no implementation for them (see above).
So in fact these methods are always overridden in the viewer itself.

Reply:
As an alternative, I can implement these functions in LLTestApp (with empty bodies)
and make them pure virtual in the base class LLApp again